queue_job/jobrunner/runner.py

```diff
@@ -146,7 +146,15 @@
 import requests
 
 import odoo
-from odoo.tools import config
+try:
+    from odoo.addons.server_environment import serv_config as config
+    if config.has_section('queue_job'):
+        queue_job_config = config['queue_job']
+    else:
+        queue_job_config = {}
+except ImportError:
+    from odoo.tools import config
+    queue_job_config = config.misc.get('queue_job', {})
```

Comment:
This is the same code than the one in `__init__.py`, could you import only `queue_job_config` from a common place?

Reply:
Done